lib/mayaUsd/ufe/UsdContextOps.cpp

```diff
@@ -100,25 +100,37 @@ class ToggleActiveStateCommand : public Ufe::UndoableCommand
 {
 public:
     ToggleActiveStateCommand(const UsdPrim& prim)
-        : _prim(prim)
     {
-        _active = _prim.IsActive();
+        _stage = prim.GetStage();
```

Comment:
Nit-pick: could have put everything in the constructor initializer list, rather than having the default constructor called, then doing the assignment:
ToggleActiveStateCommand(const UsdPrim& prim) : _stage(prim.GetStage()), _primPath(prim.GetPath()), _active(prim.IsActive) {}